nvbench/main.cuh

```diff
@@ -25,8 +25,16 @@
 #include <nvbench/option_parser.cuh>
 #include <nvbench/printer_base.cuh>
 
+#include <cstdlib>
 #include <iostream>
 
+static int _nvbench_env [[maybe_unused]] = []() -> int
+{
+  // See NVIDIA/NVBench#136
+  int retval = setenv("CUDA_MODULE_LOADING", "EAGER", 1);
```

Comment:
important: `setenv` is a POSIX extension. This change will break MSVC.

Reply:
Good catch!